### PR TITLE
Improve Nimpretty

### PR DESCRIPTION
**Pull request by juancarlospaco**
- Improve Nimpretty CLI args. Tiny diff.
- I dont think `--indent=10` or bigger is used for something useful nor in production, beyond trolling.

### CODE REVIEW

**Comment by ringabout** [comment]
probably solves https://github.com/nim-lang/Nim/issues/20078

File: nimpretty/nimpretty.nim [hunk 1]
```diff
@@ -96,6 +96,12 @@ proc main =
   if outfile.len != 0 and outdir.len != 0:
     quit "[Error] out and outDir cannot both be specified"
 
+  if opt.indWidth notin {0, 2, 4, 6, 8}:
```
Comment (Araq) [inline nimpretty/nimpretty.nim:99]: Come on, this is silly. Indentation with 3 spaces is common for some code bases and nimpretty is not nimnanny.

File: nimpretty/nimpretty.nim [hunk 2]
```diff
@@ -96,6 +96,12 @@ proc main =
...
+    quit "[Error] indent must be in {0, 2, 4, 6, 8}"
+
+  if not(opt.maxLineLen >= 80):
+    quit "[Error] maxLineLen must be >= 80"
```
Comment (Araq) [inline nimpretty/nimpretty.nim:103]: What if I want to try `maxLine=60` so that it fits on a slide which has 2 columns?

**Comment by Araq** [comment]
Tools are better when they can accept the fact that they are not Skynet.

**Pull request closed without merging**